8.0

ccarly opened the pull request:
Varias actualizaciones, probadas y testeadas.
Nuevas funcionalidades:
- Mostrar la lista de preguntas aleatoreamente.
- Vista mejoradas

ccarly committed:
Actualizado t-foreach

Se modifican las vistas de las despuestas, para que puedan mostrarse aleatorias

ccarly committed:
Se actualizan vistas

Adiciona en el en la vistas survey.question el campo random_show
Adiciona en el en la vistas survey.page para que muestre conditional y randon_show

ccarly committed:
Oculta las paginas que no cumplen con la condicion

ccarly committed:
Actualiza función get_list_questions 

Mejora en la busqueda e inicia las respuestas de no complen con la condicion

jjscarafia commented:
Buenas y muchas gracias por el pr. 
Estamos tratando de no meter cambios en la v8 ya que está estable y ya nos estamos moviendo para v9 y v10. La verdad es que vengo muy ocupado y para aceptar el PR debería probar bien que estos cambios no afecten a nuestros clientes que usan este módulo. 
Si no tenés problema, voy a ver de aceptar el PR y migrarlo a la v9 cuando migre a estos clientes (será probablemente en un par de meses).
Saludos,

ccarly commented:
Muchas gracias @jjscarafia 

Sobre el PR no creo que impacte o genere dificultades para sistemas ya en producción, en realidad son dos requerimientos que nos hicieron.

1.- Despliegues aleatorios en listas y matrices, previa configuración.
2.- Saltar las paginas de encuestas que no cumplen la condición.

Por nuestro lado mejoramos algún que otro código y vista.

Lastima que el cliente beneficiario este en la 8.0, nosotros no hicimos progresos en la 9.0 o la 10.0, por lo que nos estamos enfocando directamente en la venidera 11.0

Cualquier duda no dude en contactarnos.

ccarly committed:
Preguntas y Condiciones en la pagina

Muestra solo las preguntas que tengan condiciones o sin ella en la pagina, y comprueba que la pagina no quede en blanco para saltar a la proxima

ccarly committed:
Actualizacion de objetos
